.github/workflows/leaked_secrets_scan.yaml

```
@@ -16,7 +16,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: TruffleHog OSS
-        uses: trufflesecurity/trufflehog@ade5d91d5ca94e996377b1909a802dd9dbc51b6b
+        uses: trufflesecurity/trufflehog@v3.56.1
```

```suggestion
        uses: trufflesecurity/trufflehog@main
```

Same as https://github.com/iterative/studio/pull/7721#discussion_r1330173397